batch/batch/front_end/templates/job.html

```diff
@@ -50,6 +50,22 @@ <h2>Attempts</h2>
 {% endif %}
 
 <h2>Step Status</h2>
+<section class="progress_bar">
+  {% for step in step_statuses %}
+{% if step %}
```

Comment:
can you re-indent this file? The `if` should be two spaces further to the right than the `for`. The `for name` should then be two more spaces to the right.